governance.md

```diff
@@ -0,0 +1,129 @@
+# Principles
+
+The Confidential Containers is an open source organization and community
+that adheres to the following principles:
+
+* Open - Confidential Containers is an open source project licensed under the
+[Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0). We welcome all [contributions](https://github.com/confidential-containers/community/blob/main/CONTRIBUTING.md)
+* Respectful and welcoming - See our [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md)
+* Transparent - Whenever possible, work is done in public, either from our GitHub repositories or through public meetings. Exceptions to this include e.g. security related work that could potentially be done privately before being disclosed.
+
+# Projects
+
+The Confidential Containers organization is composed of multiple projects.
+
+A project is the primary unit of collaboration, therefore each project has its
+own repository and maintainers team. All projects follow the [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md).
+
+# Community Members and Roles
+
+Everyone is welcome to participate to the Confidential Containers projects,
+and there are 3 community member roles:
+
+## Contributor
+
+Anyone that contributed to one of the Confidential Containers projects within the
+last 12 months is a *Contributor*. Any merged Pull Request is considered a valid
+contribution.
+
+Contributions are not limited to code alone. Adding or enhancing documentation,
+tests, tools, project artifacts are all valuable ways to contribute to a
+Confidential Containers project.
+
+Project contributions will be reviewed by the project maintainers and should
+pass all applicable tests.
+
+## Maintainer
+
+Each project has one or more *Maintainer*. Project maintainers are first and
+foremost active *Contributors* to the project and are responsible for:
+
+* Setting technical directions for the project.
+* Facilitating, reviewing and merging contributions. They have write access to
+  the project repository.
+* Creating and assigning project issues.
+* Enforcing the [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md).
+
+The list of maintainers for a project is defined by the project
+`CODEOWNERS` file.
+
+### Becoming a project maintainer
+
+Existing maintainers may decide to elevate a *Contributor* to the *Maintainer*
```

Comment:
Maybe it is overkill here but perhaps highlighting that the Steering Committee can also be a route to becoming a maintainer? What if current maintainers simply don't elevate new Contributors or as can happen we end up with no active maintainers to "elevate" anyone new?